core/arch/arm/plat-stm32mp1/main.c

```diff
@@ -54,17 +54,18 @@ register_phys_mem_pgdir(MEM_AREA_IO_SEC, PWR_BASE, SMALL_PAGE_SIZE);
 register_phys_mem_pgdir(MEM_AREA_IO_SEC, RCC_BASE, SMALL_PAGE_SIZE);
 register_phys_mem_pgdir(MEM_AREA_IO_SEC, RNG1_BASE, SMALL_PAGE_SIZE);
 register_phys_mem_pgdir(MEM_AREA_IO_SEC, TAMP_BASE, SMALL_PAGE_SIZE);
+register_phys_mem_pgdir(MEM_AREA_IO_SEC, TZC_BASE, SMALL_PAGE_SIZE);
 register_phys_mem_pgdir(MEM_AREA_IO_SEC, USART1_BASE, SMALL_PAGE_SIZE);
 
 #if DDR_BASE < CFG_TZDRAM_START
 register_dynamic_shm(DDR_BASE, CFG_TZDRAM_START - DDR_BASE);
 #endif
 
-#define DRAM_END		(DDR_BASE + CFG_DRAM_SIZE)
-#define TZDRAM_END		(CFG_TZDRAM_START + CFG_TZDRAM_SIZE)
+#define _DRAM_END		(DDR_BASE + CFG_DRAM_SIZE)
```

Comment:
This doesn't seem to be related to the commit description.

Reply:
True. This is related my previous implementation where `init_stm32mp1_tzc()` was implemented in **main.c**. 
I will discard this change.